.github/workflows/deploy.yml

```
@@ -0,0 +1,37 @@
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v4
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)